backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Data/SingleRight/DelegationAccessCheckResponse/appid-508.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+    "rightKey": "ttd-am-k6/read",
+    "resource": [
+      {
+        "id": "urn:altinn:resource",
+        "value": "appid-508"
```

Comment:
Because otherwise the mocked delegationCheck will fail for them.
Since delegationCheck returns a reference to the specific resource, all the resources need unique responses.

(I didn't bother with making responses for _all_ the resources though, just the majority of the first page 😅 )